imager_images/models.py

```diff
@@ -32,71 +35,42 @@ class Photo(models.Model):
     title = models.CharField("Title", name='title', max_length=255, blank=True)
     height_field = models.IntegerField("Height", blank=True)
     width_field = models.IntegerField("Width", blank=True)
-    latitude = models.IntegerField("Latitude", blank=True)
-    longitude = models.IntegerField("Longitude", blank=True)
-    camera = models.CharField("Camera", max_length=64, blank=True)
-    lens = models.CharField("Lens", max_length=64, blank=True)
-    focal_length = models.CharField("Focal Length", max_length=32, blank=True)
-    shutter_speed = models.IntegerField("Shutter Speed", blank=True)
-    appature = models.CharField("Title", max_length=64, blank=True)
-    description = models.CharField("Title", max_length=255, blank=True)
+    description = models.CharField("Description", max_length=255, blank=True, null=True)
     date_created = models.DateField('Date Created', auto_now_add=True)
     date_modified = models.DateField('Date Modified', auto_now=True)
-    date_pub = models.DateField('Date Published', editable=True, blank=True)
+    date_pub = models.DateField('Date Published', editable=True, blank=True, null=True)
     published = models.CharField(max_length=64,
```

Comment:
If your longest "choice" is only 7 characters long, why is your `max_length` at 64?